src/RestaurantDetails.js

```diff
@@ -1,25 +1,22 @@
 import React from 'react';
 
 class RestaurantDetails extends React.Component {
-	constructor() {
```

Comment:
I think we will still need constructor and super for passing props we get from the parent